zerver/lib/email_mirror.py

```diff
@@ -231,6 +248,7 @@ def extract_and_validate(email):
     return Stream.objects.get(email_token=token)
 
 def find_emailgateway_recipient(message):
+    # type: (UserMessage) -> str
```

Comment:
I think `message` is an `email.Message`, not a Zulip UserMessage object.  I think that's true in basically all the places in this file that you have a UserMessage (with the exception of the one I note about should be a zerver.models.Message